examples/00-basic-examples/HFSS_CoordinateSystem.py

```diff
@@ -19,134 +18,150 @@
 
-#####################################################
-# Edit the Coordinate System
-# --------------------------
-# The cs1 object expose properties and methods to manipulate the coordinate system.
-# The origin can be changed
+###############################################################################
+# Modify the coordinate system.
+# -----------------------------
+# The `cs1` object exposes properties and methods to manipulate the coordinate system.
+# The origin can be changed.
+
 cs1.props["OriginX"] = 10
 cs1.props["OriginY"] = 10
 cs1.props["OriginZ"] = 10
 cs1.update()
 
-# The pointing vectors can be changed
+# The pointing vectors can be changed.
+
 ypoint = [0, -1, 0]
 cs1.props["YAxisXvec"] = ypoint[0]
 cs1.props["YAxisYvec"] = ypoint[1]
 cs1.props["YAxisZvec"] = ypoint[2]
 cs1.update()
 
-#####################################################
-# Rename the Coordinate System mode
-# ---------------------------------
-# The name can be changed
+###############################################################################
+# Rename the coordinate system.
+# -----------------------------
+
 cs1.rename('newCS')
 
-#####################################################
-# Change the Coordinate System mode
-# ---------------------------------
-# Use the option to select the mode: 0 for "Axis/Position", 1 for "Euler Angle ZXZ", 2 for "Euler Angle ZYZ"
-# This will change to "Euler Angle ZXZ"
+###############################################################################
+# Change the coordinate system mode.
+# ----------------------------------
+# Use the function `change_cs_mode` to change the mode: ``0`` for "Axis/Position", 
+# ``1`` for "Euler Angle ZXZ", or ``2`` for "Euler Angle ZYZ".
+# This command changes the mode to "Euler Angle ZXZ".
+
 cs1.change_cs_mode(1)
 
-# With the new mode there are new properties that can be edited.
+# In the new mode, there are properties that can be edited.
+
 cs1.props["Phi"] = "10deg"
 cs1.props["Theta"] = "22deg"
 cs1.props["Psi"] = "30deg"
 cs1.update()
 
-#####################################################
-# Delete the Coordinate System mode
-# ---------------------------------
-# The coordinate system can be deleted
+###############################################################################
+# Delete the coordinate system.
+# -----------------------------
+
 cs1.delete()
 
-#####################################################
-# Create a new Coordinate System defining the axis
-# ------------------------------------------------
-# All the coordinate system properties can be specified at the creation
+###############################################################################
+# Create a coordinate system, defining the axes.
+# ----------------------------------------------
+# All coordinate system properties can be specified at the creation.
+# Here the axes are specified.
+
 cs2 = hfss.modeler.create_coordinate_system(name='CS2',
```

Comment:
However, a space here is a good idea.  It's after the "section header" which describes overall what's going on in this cell.